### DISCUSSION TITLE
Welcome to OCRTOC_software_package Discussions!

### DISCUSSION THREAD

**Discussion by mdc1314** [OCRTOC/OCRTOC_software_package#20] [Announcements]
## 👋 Welcome to OCRTOC challenge !
  We’re using Discussions as a place to connect with other members of our OCRTOC community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Dujinhong02** [comment]
Will the competition be held in 2025?
